CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+**Master**
+
+* Reintroduce Gzip file generation for non-binary assets.
+
+**3.4.1** (November 25, 2015)
+
+* PathUtils::Entries will no longer error on an empty directory.
+
+**3.4.0** (October 5, 2015)
+
+* Expose method to override the sass cache in the SassProcessor.
+
+**3.3.5** (September 25, 2015)
+
+* Fix bug related to absolute path being reintroduced into history cache #141.
+
+**3.3.4** (September 1, 2015)
+
+* Relative cache contents now work with windows.
+
+**3.3.3** (August 21, 2015)
+
+* Remove more absolute paths from cache contents.
+
+**3.3.2** (August 19, 2015)
+
+* Fix cache contents to use relative paths instead of absolute paths.
+
+**3.3.1** (August 15, 2015)
+
+* Fix legacy Tilt integration when locals is required argument.
+
+**3.3.0** (August 12, 2015)
```

Comment:
Nooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo. It is really a bad idea to sync CHANGELOG for different branches. It is fine for now but I'll :fire: the CHANGELOG file later or do the same we do in rails.